src/Classes/MyRadio/CoreUtils.php

```diff
@@ -669,6 +669,8 @@ public static function checkUploadPostSize()
         } else {
             $last = '';
         }
+        $post_size = substr($post_size, 0, -1); // Remove the letter
```

Comment:
This could cause issues if $post_size doesn't have a suffix and is purely numeric. Isn't the intval call below enough? string to int casts are normally fine with ignoring trailing characters

Reply:
Tested this, it does indeed :)